### ISSUE TITLE
An error is reported due to a specific CSS：opts.node.rangeBy is not a function

### ISSUE THREAD

**Issue by hewenguang** [stylelint/stylelint#5756]
### What steps are needed to reproduce the bug?

When the code written with SCSS file contains font family, an error will be reported. as follows

```
body {
  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
}
```

The error information is as follows:

```
$ stylelint **/*.scss
TypeError: opts.node.rangeBy is not a function
    at new Warning (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/node_modules/postcss/lib/warning.js:9:29)
    at Result.warn (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/node_modules/postcss/lib/result.js:26:19)
    at report (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/lib/utils/report.js:103:9)
    at complain (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/lib/rules/max-line-length/index.js:80:4)
    at checkNewline (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/lib/rules/max-line-length/index.js:187:11)
    at /Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/lib/rules/max-line-length/index.js:73:4
    at handleMatch (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/style-search/index.js:201:5)
    at module.exports (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/style-search/index.js:189:5)
    at /Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/lib/rules/max-line-length/index.js:72:3
    at /Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/lib/lintPostcssResult.js:112:8
error Command failed with exit code 1.
```

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "extends": "stylelint-config-standard-scss",
  "overrides": [
    {
      "files": "**/*.scss",
      "rules": {
        "no-empty-source": null,
        "selector-list-comma-newline-after": null
      }
    }
  ]
}
```

### How did you run Stylelint?

npx stylelint **/*.scss

### Which version of Stylelint are you using?

14.0.1

### What did you expect to happen?

No error reported

### What actually happened?

```
$ stylelint **/*.scss
TypeError: opts.node.rangeBy is not a function
    at new Warning (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/node_modules/postcss/lib/warning.js:9:29)
    at Result.warn (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/node_modules/postcss/lib/result.js:26:19)
    at report (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/lib/utils/report.js:103:9)
    at complain (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/lib/rules/max-line-length/index.js:80:4)
    at checkNewline (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/lib/rules/max-line-length/index.js:187:11)
    at /Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/lib/rules/max-line-length/index.js:73:4
    at handleMatch (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/style-search/index.js:201:5)
    at module.exports (/Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/style-search/index.js:189:5)
    at /Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/lib/rules/max-line-length/index.js:72:3
    at /Users/ranhe/servyou/dpl-pro-javascript-template/node_modules/stylelint/lib/lintPostcssResult.js:112:8
error Command failed with exit code 1.
```

### Does the bug relate to non-standard syntax?

SCSS

**Comment by ybiquitous** [comment]
@hewenguang Thanks for using the template. Can you rewrite the issue description in English, please?

**Comment by hewenguang** [comment]
ok， updated

**Comment by hewenguang** [comment]
There are multiple postcss versions in the project. Among them, the postcss that stylelint depends on is the latest version, and the versions since other libraries in the project are lower versions, resulting in node_ Modules contains multiple postcss codes

**Comment by wangzhen-CHN** [comment]
> 项目中有多个 postcss 版本。其中stylelint依赖的postcss是最新版本，由于项目中其他库的版本都是低版本，导致node_Modules包含多个postcss代码

how to fix it? thanks

**Comment by hewenguang** [comment]
> > 项目中有多个 postcss 版本。其中stylelint依赖的postcss是最新版本，由于项目中其他库的版本都是低版本，导致node_Modules包含多个postcss代码
> 
> how to fix it? thanks

你可以在项目中明确安装一个特定的版本，如 yarn add postcss@x.x.x 或者找找依赖关系，尽量让他们以来的依赖的版本一致就可以了，不过比较麻烦

**Comment by wangzhen-CHN** [comment]
> > > 项目中有多个 postcss 版本。其中stylelint依赖的postcss是最新版本，由于项目中其他库的版本都是低版本，导致node_Modules包含多个postcss代码
> > 
> > 
> > how to fix it? thanks
> 
> 你可以在项目中明确安装一个特定的版本，如 yarn add [postcss@x.x.x](mailto:postcss@x.x.x) 或者找找依赖关系，尽量让他们以来的依赖的版本一致就可以了，不过比较麻烦

谢啦

**Comment by otavionetoca** [comment]
I was facing the same issue using postcss@8.3.6 and running npm update bumped it to 8.4.5 which then worker as expected.

**Comment by AntonyF-Andreani** [comment]
In my case following the steps fixed the problem (as suggested by @hewenguang ).
- `npm ls postcss` (that show me multiple versions of postcss)
- `npm i postcss@8.2.15` (and install one in specific)

**Comment by AndyOGo** [comment]
I face that type error again after upgrading to `v15`

> TypeError: node.rangeBy is not a function
    at Object.report (/stylelint-declaration-strict-value/node_modules/stylelint/lib/utils/report.js:35:34)
